frontend/src/Editor/Inspector/Inspector.jsx

```diff
@@ -382,15 +382,31 @@ export const Inspector = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [JSON.stringify({ showHeaderActionsMenu })]);
 
+  const handleDeleteConfirm = () => {
```

Comment:
@NotTheRightGuy Can you please wrap this handleDeleteConfirm inside a callback like below since eslint is throwing a warning error. This is because on every render, a new handleDeleteConfirm function will be created but then its is a useEffect dependency as well hence the warning. Else the PR looks good.

const handleDeleteConfirm = useCallback(() => {
    switchSidebarTab(2);
    removeComponent(component);
    setWidgetDeleteConfirmation(false);
  }, [switchSidebarTab, removeComponent, component]);

Reply:
I have wrapped the function in a callback. Please let me know if anything else needs to be changed.